.github/Dockerfile-llvm

```
@@ -0,0 +1,16 @@
+FROM alpine:3.20.3
+
+ARG RUST_VERSION=stable
+
+RUN apk add bash git cmake make g++ ninja python3 ncurses-static curl lld-dev lld lld-libs libc++-static

```

```suggestion
RUN apk add bash git cmake make ninja python3 ncurses-static curl
```

We should use the compilers from our custom built llvm. Especially gcc we don't want so we don't use it by accident.

---

The custom LLVM needs a bootstrapping compiler. I think we only want the RISC-V target in the custom LLVM build we link into resolc. If we need to provide a bootstrapping LLVM build for our supported platforms because the provided ones don't work, we should set up proper multistage LLVM builds.